privacyidea/lib/passwordreset.py

```diff
@@ -141,14 +141,8 @@ def is_password_reset():
     """
     rlist = get_resolver_list(editable=True)
     log.debug("Number of editable resolvers: {0!s}".format(len(rlist)))
-    Policy = PolicyClass()
-    policy_at_all = Policy.list_policies(scope=SCOPE.USER, active=True)
-    log.debug("Policy at all: {0!s}".format(policy_at_all))
-    policy_reset_pw = Policy.match_policies(scope=SCOPE.USER,
-                                            action=ACTION.PASSWORDRESET,
-                                            active=True)
-    log.debug("Password reset policy: {0!s}".format(policy_reset_pw))
-    pwreset = (policy_at_all and policy_reset_pw) or not policy_at_all
-    log.debug("Password reset allowed via policy: {0!s}".format(pwreset))
-
+    pwreset = Match.generic(g, scope=SCOPE.USER,
+                            action=ACTION.PASSWORDRESET,
+                            client=g.client_ip).allowed(write_to_audit_log=False)
```

Comment:
please enable audit log here

Reply:
I checked it and it leads to internal server error, since at the first moment, when the UI is called (without a login request), we do not have an audit_data available, since we do not write the mere calling of the login window to the audit log.